src/macros/middleware.rs

```diff
@@ -43,7 +44,9 @@
 /// supports annotating the macro so as to drive the inference allowing the handler
 /// code to remain with minimal annotations.
 ///
-/// ```
+/// # ignoring this, since the middleware macro needs work
+/// # TODO: migration cleanup
+/// ```rust,ignore
```

Comment:
This comment could be slightly more helpful for future readers but that's a pretty minor complaint.